docs/flow diagrams/bulk-update.md

```diff
@@ -0,0 +1,53 @@
+```mermaid
```

Comment:
Please be consistent on using the solid line for input and the dashed line for output.